packages/@o3r/extractors/builders.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://raw.githubusercontent.com/angular/angular-cli/master/packages/angular_devkit/architect/src/builders-schema.json",
+  "builders": {
```

Comment:
This builder should not be added to the project by the `ng-add` of the extractors package?

Reply:
added